example/windows/win32_window.h

```diff
@@ -40,7 +42,7 @@ class Win32Window {
   // as logical pixels and scale to appropriate for the default monitor. Returns
   // true if the window was created successfully.
   bool CreateAndShow(const std::wstring &title, const Point &origin,
-                     const Size &size);
+                     const Size &size, flutter::FlutterViewController *controller);
```

Comment:
I would like to see a limited-purpose interface (e.g., FlutterWindowDelegate or similar) passed rather than the whole controller, with only the specific methods the window needs access to.